SPECS/tpm2-abrmd/tpm2-abrmd.spec

```diff
@@ -58,9 +58,10 @@ make DESTDIR=%{buildroot} install
 %{_mandir}/man7
 
 %changelog
-* Sat May 09 00:21:38 PST 2020 Nick Samson <nisamson@microsoft.com> - 2.1.1-2
-- Added %%license line automatically
-
+*   Sun Sep 27 2020 Daniel McIlvaney <damcilva@microsoft.com> 2.3.3-1
+-   Update to 2.3.3 to solve incompatibility with tpm2-tss 2.4.0
+*   Sat May 09 00:21:38 PST 2020 Nick Samson <nisamson@microsoft.com> - 2.1.1-2
```

Comment:
While you're at it, could you modify Nick's entry to fit our format? (remove `-` before the version and update the timestamp)

Reply:
done